gatsby/content/docs/free-small-matrix-server.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Free Small Matrix Server'
```

Comment:
Maybe we could change the title to "Free Matrix Server using Oracle Cloud", or similar? Suggesting this because "Free Matrix Server" feels ambiguous - matrix.org is a "Free[ to use] Matrix Server"

Reply:
sure